hack/deployer/runner/ocp3.go

```diff
@@ -0,0 +1,199 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package runner
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	Ocp3DriverID = "ocp3"
+
+	// Field names of the SSH keys for GCP stored in Vault
+	Ocp3GCloudPrivateSSHKeyFieldName = "gcloud-ssh-private-key"
+	Ocp3GCloudPublicSSHKeyFieldName  = "gcloud-ssh-public-key"
+
+	// Ansible Docker image to manage OCP3 environments
+	AnsibleDockerImage = "eu.gcr.io/elastic-cloud-dev/ansible:53e4473"
```

Comment:
This image does not exist. I remember that it is supposed to be managed manually, but was it pushed at some point ?

Reply:
Oops yes I forgot to push it.